app/v1/models/request/AmendPensionCharges/PensionContributions.scala

```diff
@@ -18,7 +18,12 @@ package v1.models.request.AmendPensionCharges
 
 import play.api.libs.json.{Json, OFormat}
 
-case class PensionContributions(pensionSchemeTaxReference: Seq[String], inExcessOfTheAnnualAllowance: BigDecimal, annualAllowanceTaxPaid: BigDecimal)
+case class PensionContributions(pensionSchemeTaxReference: Seq[String],
+                                inExcessOfTheAnnualAllowance: BigDecimal,
+                                annualAllowanceTaxPaid: BigDecimal,
+                                isAnnualAllowanceReduced: Boolean,
```

Comment:
I believe that the 'isAnnualAllowanceReduced' is now Optional as in the [spec](https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=537428032) so this might need to be changed